### DISCUSSION TITLE
Need to Integrate the single spa cli with twin.

### DISCUSSION THREAD

**Discussion by abhiv01** [iTwin/itwinjs-core#3922] [General]
Hi,

I am trying to Integrate the single-spa cli into this Twin application.
So that I can make this application as Microfrontend .

https://single-spa.js.org/docs/create-single-spa

With the help of single spa, we can easily make application as a Microfrontend.
But I am not able to make it happen with twin.

Please help on that.

**Comment by aruniverse** [comment]
> But I am not able to make it happen 

Please elaborate, what's working and what's not working?

> I am trying to Integrate the single-spa cli into this Twin application.

The single-spa-cli is used to generate a default webpack config. As mentioned in https://github.com/iTwin/itwinjs-core/discussions/3846, you can eject from our default webpack config and see what we've added and combine the pertinent configs together.

If you had an example repo with repro steps, that would be very helpful.

**Reply by ayushibasaiwal** [reply]
@abhiv01 If you are still facing any issue can you please provide any example/ repro steps, it will help us to understand the problem better.
